index.js

```diff
@@ -101,6 +114,21 @@ const ignored = [
     property: "position",
     value: "fixed",
     ignoredProperties: ["float", "clear", "vertical-align"]
+  },
+  {
+    property: "flex-basis",
+    value: "/^.*$/",
+    ignoredProperties: ["width", "height"]
+  },
+  {
+    property: "list-style-type",
+    value: "none",
+    ignoredProperties: ["list-style-image"]
+  },
+  {
+    property: "overflow",
```

Comment:
> resize has no effect on this element because it is overflow:visible

https://github.com/captainbrosset/useless-css-properties/blob/master/rules.md#misc

---

> resize: applies to elements with overflow other than visible, and optionally replaced elements representing images or videos, and iframes

https://developer.mozilla.org/en-US/docs/Web/CSS/resize